src/game.h

```diff
@@ -981,7 +981,18 @@ class game
         timed_event_manager &timed_events; // NOLINT(cata-serialize)
         achievements_tracker &achievements();
         memorial_logger &memorial();
+
+        // Miscellaneous key/value pairs.
+        std::unordered_map<std::string, std::string> global_values;
```

Comment:
Oh god my compile times.
One, do these data members really need to be public?
Two, please bundle this into a helper class, like:
```
global_variables.h:
class global_variables {
   // All the stuff you added here
};

global_variables &get_globals();

game.h
#include "global_variables.h"
...
class game : ... {
    ...
    friend global_variables &get_globals();
    ...
    private:
    gloabal_variables global_variables_instance;
}

global_variables &get_globals() {
    return global_variables_instance;
}
```

Reply:
Then callers can just:
```
#include "global_variables.h"
...
global_variables &globvars = get_globals();
globvars.get_global_value( foo );
```